### ISSUE TITLE
Crash (1.21)

### ISSUE THREAD

**Issue by AC19970** [isXander/AdaptiveTooltips#42]
I built the latest version, but the game keeps crashing. I turned off all mods except dependencies and it still crashes.

Here's log.

[03:15:42] [main/INFO]: Loading Minecraft 1.21 with Fabric Loader 0.16.0
[03:15:42] [main/INFO]: Loading 63 mods:
	- adaptive-tooltips 1.3.0
	- fabric-api 0.101.2+1.21
	   |-- fabric-api-base 0.4.42+6573ed8cd1
	   |-- fabric-api-lookup-api-v1 1.6.68+b5597344d1
	   |-- fabric-biome-api-v1 13.0.29+5bd9f1bcd1
	   |-- fabric-block-api-v1 1.0.22+0af3f5a7d1
	   |-- fabric-block-view-api-v2 1.0.10+6573ed8cd1
	   |-- fabric-blockrenderlayer-v1 1.1.52+0af3f5a7d1
	   |-- fabric-client-tags-api-v1 1.1.15+6573ed8cd1
	   |-- fabric-command-api-v1 1.2.49+f71b366fd1
	   |-- fabric-command-api-v2 2.2.28+6ced4dd9d1
	   |-- fabric-commands-v0 0.2.66+df3654b3d1
	   |-- fabric-content-registries-v0 8.0.16+b5597344d1
	   |-- fabric-convention-tags-v1 2.0.19+7f945d5bd1
	   |-- fabric-convention-tags-v2 2.5.0+c5e2b5c6d1
	   |-- fabric-crash-report-info-v1 0.2.29+0af3f5a7d1
	   |-- fabric-data-attachment-api-v1 1.1.26+6a6dfa19d1
	   |-- fabric-data-generation-api-v1 20.2.16+16c4ae25d1
	   |-- fabric-dimensions-v1 4.0.0+6fc22b99d1
	   |-- fabric-entity-events-v1 1.6.12+6fc22b99d1
	   |-- fabric-events-interaction-v0 0.7.12+ba9dae06d1
	   |-- fabric-game-rule-api-v1 1.0.53+6ced4dd9d1
	   |-- fabric-item-api-v1 11.0.0+afdfc921d1
	   |-- fabric-item-group-api-v1 4.1.4+78017270d1
	   |-- fabric-key-binding-api-v1 1.0.47+0af3f5a7d1
	   |-- fabric-keybindings-v0 0.2.45+df3654b3d1
	   |-- fabric-lifecycle-events-v1 2.3.12+6c1df360d1
	   |-- fabric-loot-api-v2 3.0.14+3f89f5a5d1
	   |-- fabric-loot-api-v3 1.0.2+3f89f5a5d1
	   |-- fabric-message-api-v1 6.0.13+6573ed8cd1
	   |-- fabric-model-loading-api-v1 2.0.0+fe474d6bd1
	   |-- fabric-networking-api-v1 4.2.2+60c3209bd1
	   |-- fabric-object-builder-api-v1 15.1.14+d1321076d1
	   |-- fabric-particles-v1 4.0.2+6573ed8cd1
	   |-- fabric-recipe-api-v1 5.0.12+65089712d1
	   |-- fabric-registry-sync-v0 5.1.2+60c3209bd1
	   |-- fabric-renderer-api-v1 3.4.0+c705a49cd1
	   |-- fabric-renderer-indigo 1.7.0+c705a49cd1
	   |-- fabric-renderer-registries-v1 3.2.68+df3654b3d1
	   |-- fabric-rendering-data-attachment-v1 0.3.48+73761d2ed1
	   |-- fabric-rendering-fluids-v1 3.1.6+b5597344d1
	   |-- fabric-rendering-v0 1.1.71+df3654b3d1
	   |-- fabric-rendering-v1 5.0.5+df16efd0d1
	   |-- fabric-resource-conditions-api-v1 4.3.0+8dc279b1d1
	   |-- fabric-resource-loader-v0 1.3.0+56599129d1
	   |-- fabric-screen-api-v1 2.0.24+b5597344d1
	   |-- fabric-screen-handler-api-v1 1.3.86+b5597344d1
	   |-- fabric-sound-api-v1 1.0.23+6573ed8cd1
	   |-- fabric-transfer-api-v1 5.1.16+3dccd343d1
	   \-- fabric-transitive-access-wideners-v1 6.1.0+176f9036d1
	- fabricloader 0.16.0
	   \-- mixinextras 0.4.0
	- java 21
	- minecraft 1.21
	- yet_another_config_lib_v3 3.5.0+1.21-fabric
	   |-- com_twelvemonkeys_common_common-image 3.10.0
	   |-- com_twelvemonkeys_common_common-io 3.10.0
	   |-- com_twelvemonkeys_common_common-lang 3.10.0
	   |-- com_twelvemonkeys_imageio_imageio-core 3.10.0
	   |-- com_twelvemonkeys_imageio_imageio-metadata 3.10.0
	   |-- com_twelvemonkeys_imageio_imageio-webp 3.10.0
	   |-- org_quiltmc_parsers_gson 0.2.1
	   \-- org_quiltmc_parsers_json 0.2.1
[03:15:42] [main/INFO]: SpongePowered MIXIN Subsystem Version=0.8.7 Source=file:/C:/Users/{COMPUTER_USERNAME}/AppData/Roaming/com.modrinth.theseus/meta/libraries/net/fabricmc/sponge-mixin/0.15.0+mixin.0.8.7/sponge-mixin-0.15.0+mixin.0.8.7.jar Service=Knot/Fabric Env=CLIENT
[03:15:42] [main/INFO]: Compatibility level set to JAVA_21
[03:15:43] [main/WARN]: Error loading class: org/jetbrains/annotations/ApiStatus$Internal (java.lang.ClassNotFoundException: org/jetbrains/annotations/ApiStatus$Internal)
[03:15:44] [main/INFO]: Initializing MixinExtras via com.llamalad7.mixinextras.service.MixinExtrasServiceImpl(version=0.4.0).
[03:15:45] [Datafixer Bootstrap/INFO]: 226 Datafixer optimizations took 370 milliseconds
[03:15:49] [Render thread/INFO]: Environment: Environment[sessionHost=https://sessionserver.mojang.com, servicesHost=https://api.minecraftservices.com, name=PROD]
[03:15:49] [Render thread/INFO]: Setting user: {MINECRAFT_USERNAME}
[03:15:50] [Render thread/INFO]: [Indigo] Registering Indigo renderer!
[03:15:50] [Render thread/INFO]: Deserializing YACLConfig from 'C:\Users\{COMPUTER_USERNAME}\AppData\Roaming\com.modrinth.theseus\profiles\Fabric\config\yacl.json5'

**Comment by mucookul** [comment]
This doesn't look like a full log, maybe try a rebuild or you could try my jar, it works for me: https://github.com/isXander/AdaptiveTooltips/pull/41#issuecomment-2269862011

**Comment by AC19970** [comment]
> This doesn't look like a full log, maybe try a rebuild or you could try my jar, it works for me: [#41 (comment)](https://github.com/isXander/AdaptiveTooltips/pull/41#issuecomment-2269862011)

Yeah, it looks incomplete because it always ends abruptly and doesn't look like a normal terminated run at all.

**Comment by mucookul** [comment]
> > This doesn't look like a full log, maybe try a rebuild or you could try my jar, it works for me: [#41 (comment)](https://github.com/isXander/AdaptiveTooltips/pull/41#issuecomment-2269862011)
> 
> Yeah, it looks incomplete because it always ends abruptly and doesn't look like a normal terminated run at all.

WHAT, that doesn't even look like a minecraft issue but a launcher issue
